core/src/components/alert/alert.md.scss

```diff
@@ -282,3 +282,25 @@
 .alert-button-inner {
   justify-content: $alert-md-button-group-justify-content;
 }
+
+/**
+ * MD alerts should scale up to 560px x 560px
+ * on tablet dimensions.
+ */
+@media screen and (min-width: 768px) {
```

Comment:
This aligns with the iOS Card Modal breakpoints https://github.com/ionic-team/ionic-framework/blob/8227b0ee6d5250e122a34a83c644f8a74fbbafd5/core/src/components/modal/modal.ios.scss#L63

Reply:
What do we gain by doing that?

Reply:
It would be easier to keep the breakpoint consistent across components, since we wouldn't need to go reference the exact px value.

Reply:
Ok added some mixins: https://github.com/ionic-team/ionic-framework/pull/28501/commits/931c456ecb67a3183321fefbc0184af9ac66ce3a

Is this what you had in mind?

Reply:
Yeah, much better 👍